packages/coil-extension/src/background/services/Stream.ts

```diff
@@ -197,19 +206,20 @@ export class Stream extends EventEmitter {
     this._debug('aborted because stream is no longer active.')
   }
 
-  async _getBtpToken() {
-    try {
-      this._debug('fetching btp token')
-      return await this._client.refreshBtpToken(this._token)
-    } catch (e) {
-      this._debug('error fetching btp token', e.message)
-      throw e
-    }
-  }
+  async _getBtpToken(): Promise<string> {
```

Comment:
Not sure when you guys wanted to roll all this out, but could potentially develop on master behind a feature flag (WEBPACK_DEFINE_XXX) if ya felt it was more convenient in any way.